src/main/java/wooteco/subway/domain/Sections.java

```diff
@@ -0,0 +1,116 @@
+package wooteco.subway.domain;
+
+import wooteco.subway.exception.BusinessException;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Sections {
+
+    private static final int MIN_COUNT_OF_SECTION = 1;
+    private static final String NOT_EXIST_STATION = "해당 노선에 존재하는 역이 아닙니다.";
+    private static final String ALREADY_ADDED = "이미 등록되어있는 구간입니다.";
+    private static final String NO_SECTION = "추가할 수 있는 구간이 없습니다.";
+    private static final String TOO_LONG_DISTANCE = "해당 구간의 거리가 추가하려는 구간보다 더 짧습니다.";
+    private static final String CAN_NOT_DELETE = "하나의 노선에는 최소 하나의 구간이 필요합니다.";
+
+    private final List<Section> sections;
+
+    public Sections(List<Section> sections) {
+        this.sections = sections;
+    }
+
+    public LinkedList<Long> getSortedStationIds() {
```

Comment:
```suggestion
    public List<Long> getSortedStationIds() {
```

인터페이스 타입을 반환하는 것과 구현체 타입을 반환하는 것에는 어떤 차이가 있을까요?

Reply:
인터페이스 타입을 반환하는 경우 추후에 LikedList가 아닌 다른 구현체로 변화한다 하더라도 반환받는 쪽에서는 변화 없이 이용할 수 있습니다! 해당 장점을 알고있음에도 그렇게 구현하지 않는 이유는 isTerminus() 때문입니다. 해당 함수는 정렬된 지하철역의 가장 첫번째와 가장 마지막을 이용합니다. 이때 list의 get(0)와 get(list.size()-1) 로 이용하는 것이 아닌, LikedList의 getFirst() getLast()를 이용하고싶었습니다. 다른 구현체로 변화할 가능성이 없다 느껴져 더욱 이 방법을 사용했습니다. 제가 놓치고 있는 부분이 있을까요?!

Reply:
아하 제가 그 부분을 놓쳤네요. 확인했습니다!